extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -128,8 +131,34 @@ export class AmpShareTracking extends AMP.BaseElement {
    * @return {!Promise<string>}
    * @private
    */
-  getRandomBase64_() {
-    return Promise.resolve(base64UrlEncodeFromBytes(getRandomBytesArray(6)));
+  getShareTrackingRandomBase64_() {
```

Comment:
you mean no need to call `getShareTrackingRandomBase64_`, call `Promise.resolve(base64UrlEncodeFromBytes(this.getShareTrackingRandomBytes_()));` directly?